debian/build_virtualenv

```diff
@@ -36,7 +36,6 @@ esac
 dh_virtualenv \
     --install-suffix "matrix-synapse" \
     --builtin-venv \
-    --setuptools \
```

Comment:
this turns out never to have done anything; as of recent dh_virtualenv it tries to do something, which is broken.